src/EditorFeatures/CSharpTest/Completion/CompletionProviders/DeclarationNameCompletionProviderTests.cs

```diff
@@ -1620,6 +1620,121 @@ public void Method()
             await VerifyItemExistsAsync(markup, "nullables");
         }
 
+        [WorkItem(1220195, "https://github.com/dotnet/roslyn/issues/23590")]
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task TypeIsNullableStructInLocalWithNullableTypeName()
+        {
+            var markup = @"
+using System;
+
+public struct ImmutableArray<T> : System.Collections.Generic.IEnumerable<T> { }
+
+public class Class1
+{
+  public void Method()
+  {
+      Nullable<ImmutableArray<int>> $$
+  }
+}
+";
+            await VerifyItemExistsAsync(markup, "vs");
```

Comment:
Related to: https://github.com/dotnet/roslyn/issues/36364#issuecomment-501812129

@sharwell What is the expected name here?

Reply:
I'll happily chime in to say that while I may not have a strong opinion what we _should_ suggest, this suggestion is indeed confusing and nearly worthless.